frontends/concrete-python/concrete/fhe/compilation/circuit.py

```diff
@@ -270,6 +273,9 @@ def run(
                 result(s) of evaluation
         """
 
+        if self.configuration.simulate_encrypt_run_decrypt:
```

Comment:
no need to explain? like "# In simulate_encrypt_run_decrypt:, we skip encryption and decryption, and run will be normal python execution

Reply:
Same as above and below.